File: Frontend/gvpp/src/models/filterRequest.ts
export class FilterRequest {
  name: string;
  datatype: string;
  value: string;
  active: boolean;
  inputType: string;

  constructor(name: string, datatype: string, value: string, active: boolean, inputType: string) {
    this.name = name;
    this.datatype = datatype;
    this.value = value;
    this.active = active;
    this.inputType = inputType;
  }
}

File: Backend/generic-free-spots-backend/src/main/java/be/PXLResearch/code4belgium/general/DTO/FilterDTO/FilterRequest.java
package be.PXLResearch.code4belgium.general.DTO.FilterDTO;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;


@Builder
@Data
@NoArgsConstructor
@AllArgsConstructor
public class FilterRequest {
    private String name;
    private String datatype;
    private String value;
    private boolean active;
    private String inputType;
}
